plugins/gradle/gradle-plugin/src/main/java/org/wildfly/swarm/plugin/gradle/GradleDependencyResolutionHelper.java

```diff
@@ -202,10 +210,15 @@ public static boolean isProject(Project project, DependencyDescriptor descriptor
         // ------
         // Assuming that the given configuration can be resolved, get the resolved artifacts and populate the return Map.
         //
```

Comment:
The below code constructs the `Group:Artifact` coordinates for the available fractions. This is then used to determine which dependency is a fraction vs. specifically requested by the project.